acct-user/rspamd/rspamd-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for rspamd - Rapid spam filtering system"
+ACCT_USER_ID=237
+ACCT_USER_HOME=/var/lib/rspamd
```

Comment:
Oh right, you should just leave this empty. 
http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml#the-home-directory

Reply:
Thanks, I also removed unnecessary `user` eclass dependency.